guides/common/modules/con_disaster-recovery-with-active-and-passive-project-server-and-external-storage.adoc

```diff
@@ -0,0 +1,6 @@
+[id="disaster-recovery-with-active-and-passive-{project-context}-server-and-external-storage"]
+= Disaster recovery with active and passive {ProjectServer} and external storage
+
+In this scenario, you maintain two {ProjectServer}s and you store critical data externally on a network-attached storage (NAS).
```

Comment:
```suggestion
In this scenario, you maintain two {ProjectServer}s and you store critical data externally on network-attached storage (NAS).
```

I think if we were referencing a device we'd use `a network-attached storage device`.